frontend/containers/account-type-selector/account-type-item/component.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { usePress, useFocusWithin } from '@react-aria/interactions';
+import noop from 'lodash-es';
```

Comment:
Are you not importing the whole library when you do that?
```suggestion
import { noop } from 'lodash-es';
```